docs/tutorial/devtools-extension.md

```diff
@@ -1,63 +1,73 @@
 # DevTools Extension
 
-Electron supports the [Chrome DevTools Extension][devtools-extension], which can
-be used to extend the ability of devtools for debugging popular web frameworks.
+Electron supports [Chrome DevTools extensions][devtools-extension], which can
+be used to extend the ability of Chrome's developer tools for debugging
+popular web frameworks.
 
-## How to load a DevTools Extension
+## Manually loading a DevTools extension
```

Comment:
```suggestion
## How to load a DevTools extension
```

I don't think there's anything "manual" about this process, at least not more/less than it was before.

Reply:
I wanted to contrast this method vs. the `electron-devtools-installer` way of doing things (both are H2-level headings now), so I figured this one was the more "manual" approach. Do you think that would make sense to readers? I'd like to add some differentiating descriptor here.

Reply:
Ah, ok, it makes sense if this section comes _after_ the "automatic" method.